app/controllers/PayPalOneOff.scala

```diff
@@ -27,12 +28,14 @@ class PayPalOneOff(
     paymentAPIService: PaymentAPIService,
     identityService: IdentityService,
     settingsProvider: AllSettingsProvider,
-    tipMonitoring: Tip
+    tipMonitoring: Tip,
+  stage: Stage
```

Comment:
is this indentation alright? looks underindented on github diff

Reply:
i think my intellij messed up spaces/tabs, will go over today